src/components/Tab/Tab.jsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './Tab.scss';
+import classnames from 'clsx';
+import {Typography} from '../Typography';
+
+export const tabSizes = ['small', 'default', 'big'];
```

Comment:
As I can see the mockup just provide the medium size

Reply:
updated